main.py

```diff
@@ -0,0 +1,12 @@
+import settings
+import os
+import subprocess
+import utils
+from time import sleep
+
+while True:
+    utils.init_datadog()
+    utils.Report.ok()
+    subprocess.Popen(
```

Comment:
Agreed, and the Datadog library is reinitialised inside the subprocess.

Reply:
Seem like it uses twisted under the hood ... so might not be simplest library import without the helper runner, but I'll have a poke around.

I think another issue with this loop is that Popen is async ... so as it stands it will run a new process every 5mins regardless of whether the last one complete ... which could be leak (although unlikely given the 5min timer).

That first `init_datadog` call is just to allow for the `Report.ok()` to phone in I believe ... not the main reporting.

Reply:
I think it would be best if `utils.Report.ok()` was only called if we successfully got all the data. But at present there's no easy way to code that.